ac

```diff
@@ -230,9 +230,9 @@ ac_bandit(){
 # ------------------------------------------------------------------------------
 #->ac-build:
 ## Build and install collection of the local GH branch.
-## Usage: ac [-b, --ac-build]
+## Usage: ac [-ac-lint]
```

Comment:
shouldn't this be 2 dashes?  i.e.: Usage: ac [--ac-lint] ?

Reply:
you are correct and it is now updated.